<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>EditorTexto</class>
 <widget class="QMainWindow" name="EditorTexto">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>559</width>
    <height>484</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>EditorTexto</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QPlainTextEdit" name="CampoTexto">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>140</y>
      <width>521</width>
      <height>241</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="SaveButton">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>30</y>
      <width>99</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text">
     <string>Save File</string>
    </property>
   </widget>
   <widget class="QPushButton" name="InfoButton">
    <property name="geometry">
     <rect>
      <x>450</x>
      <y>390</y>
      <width>99</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text">
     <string>Info</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="file_name">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>30</y>
      <width>371</width>
      <height>27</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="Label_File_Name">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>0</y>
      <width>81</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>File Name:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="path_name">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>80</y>
      <width>371</width>
      <height>27</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>70</y>
      <width>101</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Path Not Work</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>559</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="StatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>StatusBar</class>
   <extends>QStatusBar</extends>
   <header>statusbar.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
